design/object-bucket/object-bucket.md

```diff
@@ -0,0 +1,183 @@
+# ObjectBucketClaim CRD and Controller
+
+## Proposed Feature
+
+**A generalized method of S3 bucket provisioning through the implementation of an ObjectBucketClaim CRD and controller.** 
+
+Users will request buckets via the Kubernetes API and have returned ConfigMaps with connection information for that bucket.  
```

Comment:
keep all singular or all plural. Eg. 
_Users will request a bucket via the Kubernetes API and have returned a ConfigMap with connection information for that bucket_ or
_Users will request buckets via the Kubernetes API and have returned ConfigMaps with connection information for each bucket. _